Extensions/xkit_patches.js

```diff
@@ -441,6 +441,36 @@ XKit.tools.add_function = function(func, exec, addt) {
 	}
 };
 
+XKit.tools.unsafe_eval = function(f, context) {
```

Comment:
This is definitely a slightly fancier version of https://github.com/new-xkit/XKit/blob/master/Extensions/xkit_patches.js#L215. I really like its use of removeChild after the appendChild, that's something we should definitely copy over. Otherwise, I don't think we should have two functions that do the same thing.